hw/hal/include/hal/hal_gpio.h

```diff
@@ -101,6 +101,15 @@ int hal_gpio_init_in(int pin, hal_gpio_pull_t pull);
  */
 int hal_gpio_init_out(int pin, int val);
 
+/**
+ * Deinitialize the specified pin to revert the previous initialization
```

Comment:
I think this should say _to revert to default configuration_ as _previous initialization_ may suggest this will revert to previous call to `hal_gpio_init`

Reply:
Agree, changed by: https://github.com/apache/mynewt-core/pull/1630/commits/9acf80b398984ac1164c9cb7fc387f28e1b275ee